src/pybrake/middleware/flask.py

```diff
@@ -167,10 +168,19 @@ def _sqla_before_cursor_execute(
 
 def _after_cursor(config):
     def _sqla_after_cursor_execute(
-        conn, cursor, statement, parameters, context, executemany
+            conn, cursor, statement, parameters, context, executemany
```

Comment:
Why this change, though?

Reply:
It's done by the pycharm formatter. I'll remove it before merge.